packages/next/build/webpack/config/blocks/css/index.ts

```diff
@@ -86,6 +23,94 @@ const regexCssModules = /\.module\.css$/
 const regexSassGlobal = /(?<!\.module)\.(scss|sass)$/
 const regexSassModules = /\.module\.(scss|sass)$/
 
+let postcssInstancePromise: Promise<any>
+async function lazyPostCSS(ctx: ConfigurationContext) {
```

Comment:
I hope this is only ever called with the same ConfigurationContext. Otherwise this need to cache per `ctx`

Reply:
The options passed are always the same, I've changed it to be explicit about the options instead of passing the full ctx 👍